src/haddock/gear/prepare_run.py

```diff
@@ -807,40 +821,48 @@ def copy_molecules_to_data_dir(
         Whether to preprocess input molecules. Defaults to ``True``.
         See :py:mod:`haddock.gear.preprocessing`.
     """
-    topoaa_dir = zero_fill.fill("topoaa", 0)
+    # Removes digit from module name
+    # Build regex to capture '<name>.<digit>'
+    name_digit_regex = re.compile(r"(\w+)\.\d+")
```

Comment:
This will not capture with the module name does not have a digit, is that the intended behaviour? If you want to capture names that might not have the `.`, the regex should be `(\w+)(\.\d+)?`

Reply:
All of them have a '.' at this stage, separating `module name` and `module step`.
This is performed/ensured earlier by `load()` from `haddock.gear.config`.
But indeed, your regex is more flexible.